plexapi/library.py

```diff
@@ -1084,7 +1084,38 @@ def __len__(self):
 
 
 @utils.registerPlexObject
-class Collections(PlexObject):
+class Collections(PlexPartialObject):
```

Comment:
Was this changed to PlexPartialObject to allow a reload?

Reply:
9685b57   
> this will allow collections to use the base.PlexPartialObject.edit method

More so that I can use the `.edit()`